tools/gcp/stress_test/stress_test_utils.py

```diff
@@ -107,12 +107,22 @@ def check_if_any_tests_failed(self, num_query_retries=3):
     query = ('SELECT event_type FROM %s.%s WHERE run_id = \'%s\' AND '
              'event_type="%s"') % (self.dataset_id, self.summary_table_id,
                                    self.run_id, EventType.FAILURE)
-    query_job = bq_utils.sync_query_job(self.bq, self.project_id, query)
-    page = self.bq.jobs().getQueryResults(**query_job['jobReference']).execute(
-        num_retries=num_query_retries)
-    num_failures = int(page['totalRows'])
-    print 'num rows: ', num_failures
-    return num_failures > 0
+    try:
+      query_job = bq_utils.sync_query_job(self.bq, self.project_id, query)
+      page = self.bq.jobs().getQueryResults(
+          **query_job['jobReference']).execute(num_retries=num_query_retries)
+      num_failures = int(page['totalRows'])
+      print 'num rows: ', num_failures
+      return num_failures > 0
+    # TODO (sreek): Cleanup the following lines once we have a better idea of
```

Comment:
Unfortunately yes ;). 
Clearly the response doesn't have `totalRows` but I don't know why. Based on what I could find on some forums, it might be a timeout issue that manifests this way - but I am not really sure. I am printing the whole response i.e `page` to see if I get a better idea. It hasn't repro'ed on my machine (yet) - So far, this failure has been happening approximately 1 out of 4 times when running the stress tests for ~2hrs on jenkins